dotnet-datetime/DateTimeOffsetVsDateTime/DateTimeOffsetVsDateTime/Program.cs

```diff
@@ -0,0 +1,26 @@
+﻿var dateTime = DateTime.Now;
+Console.WriteLine($"DateTime: {dateTime}"); 
+var dateTimeOffset = DateTimeOffset.Now;
+Console.WriteLine($"DateTimeOffset: {dateTimeOffset}"); 
+
+//TimeZone vs Offset 
+static List<TimeZoneInfo> GetTimeZoneFromOffset(TimeSpan offset) =>
+    TimeZoneInfo.GetSystemTimeZones()
+    .Where(tz => tz.BaseUtcOffset == offset)
+    .ToList();
```

Comment:
There should be a newline after this to reflect the suggestion from the Senior Editor.